Assets/Scripts/EntityBehaviour.cs

```diff
@@ -0,0 +1,46 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EntityBehaviour : MonoBehaviour
+{
+    [SerializeField] private int enemyType;
+
+    [SerializeField] private int health;
+    
+
+    public void Trigger_TakeDamage(int _bulletType)
+    {
+        //TAKE DAMAGE
+        if (_bulletType == enemyType)
+            Trigger_InstantKill();
+
+        else health--;
```

Comment:
this would be prettier with the optional `{}` i think

Reply:
a little hard to read, as we've mixed and matched whether the statement goes on the same line or the one after the control flow expression